utils/wasm-gen/src/generator/syscalls/invocator.rs

```diff
@@ -143,6 +143,20 @@ impl<'a, 'b> From<DisabledAdditionalDataInjector<'a, 'b>> for SyscallsInvocator<
 }
 
 impl<'a, 'b> SyscallsInvocator<'a, 'b> {
+    /// Returns the size of the memory in pages.
+    fn memory_size_pages(&self) -> u32 {
+        self.module
+            .initial_mem_size()
+            // To instantiate this generator, we must instantiate SyscallImportsGenerator, which can be
+            // instantiated only with memory import generation proof.
+            .expect("generator is instantiated with a memory import generation proof")
+    }
+
+    /// Returns the size of the memory in bytes.
+    fn memory_size_in_bytes(&self) -> u32 {
```

Comment:
maybe like this as `memory_size_pages` is done?

```suggestion
    fn memory_size_bytes(&self) -> u32 {
```